nav_msgs/OccupancyGrid occ_map
geometry_msgs/Point[] reference_line
